src/System/Wlog/LogHandler/Simple.hs

```diff
@@ -122,6 +114,7 @@ fileHandler :: FilePath -> Severities -> IO (GenericHandler Handle)
 fileHandler fp sev = do
     createDirectoryIfMissing True (takeDirectory fp)
     h <- openFile fp ReadWriteMode
+    hSetEncoding h utf8
```

Comment:
After testing this, it's not sufficient to set the encoding on the filehandler only since it fails when logging to the console.

If put line `102` instead, as:

```hs
hSetEncoding privData utf8
```

It works fine (note that the `streamHandler` is invoked by the `fileHandler` so the encoding is set in both cases.